controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook_test.go

```diff
@@ -221,6 +221,21 @@ func TestKubeadmControlPlaneValidateUpdate(t *testing.T) {
 			ImageTag: "v9.1.1",
 		},
 	}
+
+	etcdLocalImageBuildTag := before.DeepCopy()
+	etcdLocalImageBuildTag.Spec.KubeadmConfigSpec.ClusterConfiguration.Etcd.Local = &kubeadmv1beta1.LocalEtcd{
+		ImageMeta: kubeadmv1beta1.ImageMeta{
+			ImageTag: "v9.1.1_invalidBuild1",
```

Comment:
nit: Remove `invalid` from here since this is a valid example. Got a little confused for a sec. 🙂

Reply:
Oops